core/src/main/java/com/sequenceiq/cloudbreak/conf/MailSenderConfig.java

```diff
@@ -100,7 +104,11 @@ private String missingVars() {
 
     private Properties getJavaMailProperties() {
         Properties props = new Properties();
-        props.put("mail.transport.protocol", "smtp");
+        if (useSmtps) {
```

Comment:
why we need this boolean?

wouldn't be simpler just to pass the transport protocol like smtps or smtp. The default would be smtp